lib/stripe/subscriptions/usage.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Stripe.SubscriptionItem.Usage do
+  @moduledoc """
+  Work with Stripe usage record objects.
+
+  Stripe API reference: https://stripe.com/docs/api/usage_records
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type t :: %__MODULE__{
+          id: Stripe.id(),
+          object: String.t(),
+          livemode: boolean,
```

Comment:
So it looks like there are two structs - `Usage` && `UsageSummary`, the latter having a field `invoice: String.t() | nil`.  I think we could just use one struct and add `invoice`.  What do you think?

Reply:
yeah, I was looking at this too, but following how things were setup, I was thinking some might want it treated differently. So was trying to stay close to how Stripe has it setup

Reply:
good point, adding this in